plugins/editors/tinymce/tinymce.php

```diff
@@ -865,32 +867,32 @@ protected static function getGlobalFilters()
 					$filter->blockedAttributes = $customListAttributes;
 				}
 			}
-			// Blacklists take second precedence.
-			elseif ($blackList)
+			// DisallowList take second precedence.
```

Comment:
```suggestion
			// Disallow List take second precedence.
```
Formalise the wording to show as separate words.